addons/default/visiosoft/base-theme/resources/config/settings/settings.php

```diff
@@ -71,4 +71,24 @@
     'style' => [
         'type' => 'anomaly.field_type.editor',
     ],
+
+	'default_country' => [
+		'bind' => 'visiosoft.theme.base::countries.enabled',
+		'env' => 'ADV_ENABLED_COUNTRIES',
+		'type' => 'anomaly.field_type.select',
+		'required' => false,
+		'config' => [
+			'default_value' => function () {
+				return [config('visiosoft.theme.base::countries.default')];
+			},
+			'options' => function () {
+				$array = CountryModel::query()->get();
+				$cur = array();
+				foreach ($array as $key => $value) {
+					$cur[$value['abv']] = $value['name'];
+				}
```

Comment:
why not use pluck here?

Reply:
https://github.com/openclassify/openclassify/pull/715/commits/3220976fd2b79f320c2cf9ec5eab1bc6adc01ca6